.github/workflows/checks.yml

```diff
@@ -14,7 +14,7 @@ jobs:
           submodules: true
       - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22
```

Comment:
How come there are `22` bumps for adding `24` tests?

Reply:
24 being released makes 22 the "active" LTS release